help/configure-notification-bot.md

```diff
@@ -69,6 +69,27 @@ these messages is "signups".
 
 {end_tabs}
 
+### Zulip update announcements
+
+In Zulip, you can get notified about significant changes
+to the Zulip user experience such as new features, new keyboard shortcuts,
+changes in terminology, etc.
+
+You can configure where the Zulip notification bot announces Zulip news,
+or disable the Zulip news announcement messages entirely. The topic for
+these messages is "Zulip updates".
```

Comment:
Let's consistently call this "updates". Something like this:

> You can configure where the Zulip notification bot sends Zulip updates,
or disable the Zulip update messages entirely. The topic for
these messages is "Zulip updates".